pghoard/transfer.py

```diff
@@ -174,19 +176,14 @@ def increment(self, file_key: str, total_bytes_uploaded: float) -> None:
                             file_key,
                             stalled_age,
                         )
-                metric_data = {
```

Comment:
Removed this because we already monitor it under the metric name pghoard.total_upload_size for all file types (xlog, basebackup_chucks....)
 https://github.com/Aiven-Open/pghoard/blob/a1da4469e3dbad833a260d52f680facc03e2c367/pghoard/transfer.py#L411